back-end/sports_platform/sports_platform_api/models/badge_models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+class Badge(models.Model):
+    class Meta:
+        db_table = 'badge'
+
+    name = models.CharField(primary_key=True, max_length=30)
+    wikidata = models.CharField(max_length=30, blank=True)
+
+    @staticmethod
+    def get_badges():
+        try:
+            badges = Badge.objects.all()
+
+            badges_res = []
+            for badge in badges:
+                item = {}
+                if badge.wikidata:
+                    item['@context'] = "https://www.wikidata.org/entity" + badge.wikidata
```

Comment:
When we give a link this, is this a valid one? It is weird to me with path "entityQ...." but I do not know, I am just asking.

Reply:
I guess there should be "/" after "entity"

Reply:
Fixed